services/api-server/src/simcore_service_api_server/models/schemas/files.py

```diff
@@ -25,6 +25,16 @@ class FileMetadata(BaseModel):
     # SEE https://ant.apache.org/manual/Tasks/checksum.html
     checksum: str = Field(None, description="MD5 hash of the file's content")
 
+    class Config:
+        schema_extra = {
+            "example": {
+                "file_id": "f0e1fb11-208d-3ed2-b5ef-cab7a7398f78",
+                "filename": "Architecture-of-Scalable-Distributed-ETL-System-whitepaper.pdf",
+                "content_type": "application/pdf",
```

Comment:
one question.. this is going to the client? are you sure about the snake casing? I actually have no clue but if I search:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type. just wondering. but if FastApi already does the conversion, then forget that comment.

Reply:
I am using built in [mimetype](https://docs.python.org/3/library/mimetypes.html#mimetypes.guess_type) module to deduce it so it should be standard. See [here](https://github.com/ITISFoundation/osparc-simcore/blob/c2c17fbf41fca8cecbca4e1b44b13c0ffe9a395f/services/api-server/src/simcore_service_api_server/api/routes/files.py#L56)

Reply:
Aha .. I understand you now. you mean the field-names! in any case, this content_type is NOT used in the header but as a body of the metadata.